lib/charms/operator_libs_linux/v2/snap.py

```diff
@@ -332,19 +333,17 @@ def get(self, key: Optional[str], *, typed: bool = False) -> Any:
 
         return self._snap("get", [key]).strip()
 
-    def set(self, config: Dict[str, Any], *, typed: bool = False) -> str:
+    def set(self, config: Dict[str, Any], *, typed: bool = False) -> None:
```

Comment:
Technically this is a breaking change, but it previously returned whatever progress garbage that `snap set` printed to stdout, so charmers almost certainly weren't using it. So we're okay with changing this type to None. @james-garner-canonical to double check charm usage.

Reply:
Changed to private, thanks!